apps/studio/data/database-queues/database-queues-toggle-postgrest-mutation.ts

```diff
@@ -192,6 +192,22 @@ grant all privileges on all tables in schema pgmq to postgres, service_role;
 alter default privileges in schema pgmq grant all privileges on tables to postgres, service_role;
 
 grant usage on schema pgmq to postgres, anon, authenticated, service_role;
+
+
+/*
+  Grant access to sequences to API roles by default. Existing table permissions
+  continue to enforce insert restrictions. This is necessary to accomodate the
+  on-backup hook that rebuild queue table primary keys to avoid a pg_dump segfault.
+  This can be removed once logical backups are completely retired.
+*/
+grant usage, select, update
+on all sequences in schema pgmq
+to anon, authenticated;
```

Comment:
@olirice we don't have to do this for `service_role`?

Reply:
thanks, yes. updated